packages/next/test/unit/fixtures/01-normalize-routes/yarn.lock

```diff
@@ -0,0 +1,184 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@next/env@12.3.1-canary.0":
```

Comment:
Should this lockfile be committed? We don't commit it for other fixtures to continue testing new canaries.

Reply:
Removing, 1 sec